.github/workflows/fcos-build.yml

```diff
@@ -0,0 +1,34 @@
+name: FCOS Build from ${{ env.COPR_OWNER }}/${{ env.COPR_PROJECT }}
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  COPR_OWNER: "rhcontainerbot"
+  COPR_PROJECT: "podman-next"
+
+jobs:
+  wait-for-task:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up wait-for-copr
+      run: |
+        pip3 install git+https://github.com/packit/wait-for-copr.git@main
+
+    - name: Wait for ${{ env.COPR_OWNER }}/${{ env.COPR_PROJECT }} build
+      run: |
+        # wait-for-copr attempts 180 tries with 10s between each try by default
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman `git rev-parse --short HEAD`
+        echo "COPR build ready!"
+
+    - name: Build FCOS image
+      run: |
+        git clone https://github.com/coreos/layering-examples
+        cd layering-examples/podman-next
+        podman build .
```

Comment:
This is really confusing, it probably deserves a comment(s) at least, and likely quite a few additional steps:

* Why does the script check out a repo. when the `checkout-action` is designed for this purpose (and properly deals with various aspects in a predictable way)?
* Why does the workflow check out the podman repo. a few steps above, only to (apparently) not use any of that content?
* Why is an "example" build context used here?
* Are there very clear/obvious docs/comments in `https://github.com/coreos/layering-examples` under `layering-examples/podman-next` pointing at ***this*** workfow, and warning/cautioning any developers that changes will impact automation outside the repo?
* Can the "example" code be moved and maintained in the podman repo. instead (perhaps somewhere under `contrib/...`?
* If the build succeeds but produces a "bad" output image, how will anybody know.  Can you add steps/scripts that do some testing/checking of the produced image to confirm it's valid?
* Similarly, if the build fails nobody will notice for potentially a long time.  This step should have a separate on-failure e-mail notification setup - which begs the same workflow-frequency spam issue mentioned above.

Reply:
To answer your main q's, The Containerfile mentioned in that repo directory was directly building fcos images using podman-next, and it seems to be actively maintained as well as what podman-desktop people are looking for. That's why the direct usage. That step is in no way complete in its current form. I guess we could add them to our own `contrib/` and answer the other q's you have and maybe have something for status reporting (don't know what that is but maybe fcos / podman-desktop can help us here).

Reply:
Thanks for the background.  To try and clarify, the main things that bugged me are: The "example" in the name signifies (for me) it can be modified w/o much regard to other stuff.  And presumably (I didn't look) no docs/comments warning about any modifications.  In other words, it would be easy for someone to change the `Containerfile` in a way that affected this job, w/o any way of anybody knowing that until potentially much later.

Does that make sense?